app/presenters/bidding_status_presenter/available.rb

```diff
@@ -19,6 +19,26 @@ def label
     'Open'
   end
 
+  def bid_label(user)
+    bid = highlighted_bid(user)
+    amount = Currency.new(bid.amount)
+
+    if bid.is_a?(NullBid)
+      I18n.t('bidding_status.available.bid_label.no_bids')
+    elsif auction.type == 'sealed_bid'
+      I18n.t('bidding_status.available.bid_label.sealed_bid',
```

Comment:
that's an interesting idea! And good point about presenter logic. Could be a task for later if you feel it's outside the scope of this PR